06-reactJs/components/form.js

```diff
@@ -0,0 +1,48 @@
+var React = require('react');
+
+var Form = React.createClass({
+    getInitialState: function () {
+        return({
+            title: '',
+            genre: '',
+            favorite: false
+        });
+    },
+    render: function () {
+        return (
+            <form onSubmit={this.handleSubmit}>
+                <input type='text' placeholder='Title' refs='title' onChange={this.handleTitleChange} />
```

Comment:
**Minor:** when defining properties on HTML tags, we use double quotes (I know, kinda confusing, but it's the only case besides JSON files where we use double quotes. :P).